sources/HUBUtilities.h

```diff
@@ -51,6 +52,23 @@ static inline BOOL HUBPropertyIsEqual(NSObject * _Nullable objectA, NSObject * _
 }
 
 /**
+ *  Return whether an object conforms to a protocol
+ *
+ *  @param object The object
+ *  @param aProtocol The protocol
+ *
+ *  This function allows checking protocol conformance without inheriting from NSObject.
+ */
+static inline BOOL HUBConformsToProtocol(id _Nullable object, Protocol *aProtocol)
```

Comment:
I don't see any tests for `HUBConformsToProtocol`. Can we add some, considering it's now an important part of our codebase?